staging/src/k8s.io/api/admission/v1alpha1/types.go

```diff
@@ -48,7 +47,7 @@ type AdmissionReviewSpec struct {
 	// +optional
 	OldObject runtime.RawExtension `json:"oldObject,omitempty" protobuf:"bytes,3,opt,name=oldObject"`
 	// Operation is the operation being performed
-	Operation admission.Operation `json:"operation,omitempty" protobuf:"bytes,4,opt,name=operation"`
+	Operation Operation `json:"operation,omitempty" protobuf:"bytes,4,opt,name=operation"`
```

Comment:
@liggitt is this going to mess with proto encoding?  I think they were both strings in the end.